tasks.py

```diff
@@ -0,0 +1,308 @@
+"""Call various Terraform actions."""
+import os
+import os.path
+from datetime import datetime, timezone
+from hashlib import sha256
+from time import sleep
+
+import requests
+from invoke import task, call
+from invoke.exceptions import Exit
+
+from kin import KinClient, Environment as KinEnvironment
+from kin.blockchain.builder import Builder
+import kin_base
+
+
+GLIDE_ARCH = 'linux-amd64'
```

Comment:
dont forget to merge my PR later to make it for mac as well

Reply:
:+1: